packages/frontend/editor-ui/src/views/Evaluations/EvaluationsRootView.vue

```diff
@@ -0,0 +1,146 @@
+<script setup lang="ts">
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { useUsageStore } from '@/stores/usage.store';
+import { useAsyncState } from '@vueuse/core';
+import { PLACEHOLDER_EMPTY_WORKFLOW_ID } from '@/constants';
+import { useCanvasOperations } from '@/composables/useCanvasOperations';
+import { useToast } from '@/composables/useToast';
+import { useI18n } from '@/composables/useI18n';
+import { useRouter } from 'vue-router';
+import { useEvaluationStore } from '@/stores/evaluation.store.ee';
+import { computed } from 'vue';
+import { N8nLink, N8nText } from '@n8n/design-system';
+import EvaluationsPaywall from '@/components/Evaluations/Paywall/EvaluationsPaywall.vue';
+import SetupWizard from '@/components/Evaluations/SetupWizard/SetupWizard.vue';
+
+const props = defineProps<{
+	name: string;
+}>();
+
+const workflowsStore = useWorkflowsStore();
+const usageStore = useUsageStore();
+const evaluationStore = useEvaluationStore();
+const router = useRouter();
+const toast = useToast();
+const locale = useI18n();
+
+const { initializeWorkspace } = useCanvasOperations({ router });
+
+const evaluationsLicensed = computed(() => {
+	return usageStore.workflowsWithEvaluationsLimit !== 0;
+});
+
+const showWizard = computed(() => {
+	const runs = Object.values(evaluationStore.testRunsById ?? {}).filter(
+		({ workflowId }) => workflowId === props.name,
+	);
+	return runs.length === 0;
+});
+
+// Method to run a test - will be used by the SetupWizard component
+async function runTest() {
+	await evaluationStore.startTestRun(props.name);
+	await evaluationStore.fetchTestRuns(props.name);
+}
```

Comment:
this needs some error handling, also duplicated across EvaluationsView, maybe it should be a composable (optional suggestion).